tomato-timer/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "tomato-timer",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "react-google-login-component": "^0.7.0"
```

Comment:
Thanks for pointing this guy out @sta7ey 
@Starpit390 make sure to include in the title or the description next time that you have added google login and that was part of the objective of the work in the PR